packages/calling-stateful-client/src/Converter.ts

```diff
@@ -85,7 +85,9 @@ export function convertSdkParticipantToDeclarativeParticipant(
     callEndReason: participant.callEndReason,
     videoStreams: declarativeVideoStreams,
     isMuted: participant.isMuted,
-    isSpeaking: participant.isSpeaking
+    isSpeaking: participant.isSpeaking,
+    /* @conditional-compile-remove(raise-hands) */
+    raisedHand: undefined // TODO: [raise-hand] add raise hand state
```

Comment:
Will this be available once the communication-calling package RemoteParticipant has a "raiseHand" prop?